lib/config.js

```diff
@@ -24,8 +24,12 @@ module.exports.getConfig = async function getConfig({ context, configName }) {
       log({
         context,
         message:
-          'Config validation errors, please fix the following issues in release-drafter.yml:\n' +
-          joiValidationErrorsAsTable(error),
+          'Config validation errors, please fix the following issues in ' +
+          (configName || DEFAULT_CONFIG_NAME) +
+          ':\n' +
+          joiValidationErrorsAsTable(error) +
+          '\nNote: This message will also be output if the configuration file is not found. ' +
```

Comment:
Should be possible to say `if (repoConfig == null)` ?

Reply:
I tried it, and it seems that `repoConfig` does not become null even if the config file does not exist.